### ISSUE TITLE
[BUG] "Failed to register FFmpeg libraries" on startup (fresh install)

### ISSUE THREAD

**Issue by Oyro** [robvdpol/RaceControl#213]
**Describe the bug**
Installed .NET Runtime 5.0.9 (x64). Installed Race Control 2.3.5. Error appears when staring Race Control during the logo-screen. I have also tried previous versions of Race Control with no success.

I have checked that the path provided to flyleaf ("C:\Program Files\Race Control\FFmpeg") in fact do contain all the FFmpeg dll's.

**Screenshots**
![RaceControllError](https://user-images.githubusercontent.com/20801112/131215458-d468a787-ca9b-4a92-941f-962bd875678f.png)

**Desktop (please complete the following information):**
 - Windows 10 Pro 21H1 64bit
 - Race Control 2.3.5
 - .NET Runtime 5.0.9 (x64)

**Additional context**
2021-08-28 11:42:41.9220 ERROR An unhandled exception occurred.
System.Exception: Failed to register FFmpeg libraries
 ---> System.DllNotFoundException: Unable to load DLL 'avformat.58 under C:\Program Files\Race Control\FFmpeg': The specified module could not be found.
   at FFmpeg.AutoGen.ffmpeg.LoadLibrary(String libraryName, Boolean throwException)
   at FFmpeg.AutoGen.ffmpeg.<>c.<.cctor>b__6_0(String libraryName)
   at FFmpeg.AutoGen.ffmpeg.<>c.<.cctor>b__6_342()
   at FlyleafLib.Master.RegisterFFmpeg(String absolutePath, Int32 verbosity)
   --- End of inner exception stack trace ---
   at FlyleafLib.Master.RegisterFFmpeg(String absolutePath, Int32 verbosity)
   at RaceControl.App.Initialize() in D:\Projects\RaceControl\RaceControl\RaceControl\App.xaml.cs:line 63
   at Prism.PrismApplicationBase.InitializeInternal()
   at Prism.PrismApplicationBase.OnStartup(StartupEventArgs e)
   at RaceControl.App.OnStartup(StartupEventArgs e) in D:\Projects\RaceControl\RaceControl\RaceControl\App.xaml.cs:line 56
   at System.Windows.Application.<.ctor>b__1_0(Object unused)
   at System.Windows.Threading.ExceptionWrapper.InternalRealCall(Delegate callback, Object args, Int32 numArgs)
   at System.Windows.Threading.ExceptionWrapper.TryCatchWhen(Object source, Delegate callback, Object args, Int32 numArgs, Delegate catchHandler)

**Comment by SuRGeoNix** [comment]
Possible RaceControl doesn't have access to the path. Try to run RaceControl as Administrator.

**Comment by Oyro** [comment]
I have tried running Race Control as admin. Unfortunately same error.

**Comment by robvdpol** [comment]
Could you please try the portable release to see whether it has the same problem?

**Comment by Oyro** [comment]
Hello! I checked out the portable 2.4.0 version, same issue.

**Comment by maantje** [comment]
Same error here. App used to work fine, reinstalled windows and now getting this error when trying to start RaceControl. 

Portable version also does not work here.

**Comment by maantje** [comment]
@Oyro I managed to get it to work, can you check if this fixes it for you too?

Press Windows key -> type "Manage optional features" -> press Enter key -> click "Add a feature" -> Add Media Feature Pack -> Reboot.

After rebooting Race Control opens correctly again for me.

**Comment by Oyro** [comment]
> Press Windows key -> type "Manage optional features" -> press Enter key -> click "Add a feature" -> Add Media Feature Pack -> Reboot.

@maantje This fixed the issue for me! I had no idea "Feature" settings were a thing in windows... Thanks!

**Comment by maantje** [comment]
@Oyro Great to hear you got it work! I think this issue should stay open until we updated the readme or fixed it another way.

I did some more digging and and we got this issue because we installed a N version of Windows which does not come by default with the Media Feature Pack. 

@robvdpol Should we add a step in the Manual installation section about installing the [Media Feature Pack](https://support.microsoft.com/en-us/topic/media-feature-pack-for-windows-10-n-may-2020-ebbdf559-b84c-0fc2-bd51-e23c9f6a4439)?
